screen-access-advanced.ks.in

```diff
@@ -48,7 +48,7 @@ echo "${generated_config}" > etc/sysconfig/anaconda
 %post --nochroot --interpreter /bin/bash
```

Comment:
Please describe what you are fixing here in a few words in the commit message.

Reply:
Fixed.